apyros/metaopen.py

```diff
@@ -0,0 +1,17 @@
+"""
+   only wrapper for ZIP files
+"""
+
+import os
+from zipfile import ZipFile
+
+
+def metaopen(filename, mode='r'):
+    """Wrapper for opening files optionally from ZIP"""
+    assert mode in ['r', 'rb'], mode  # just to know what modes we use
+    if '.zip' in filename:
+        return ZipFile.open(ZipFile(os.path.dirname(filename)), os.path.basename(filename))
```

Comment:
:-) OK, thanks - that was my first misunderstanding of ZipFile.open() ... fixed, rebased, merged. Thanks.